src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj

```diff
@@ -36,6 +36,7 @@ This package is an internal implementation of the .NET Core SDK and is not meant
 
     <!-- Reference implementation assemblies in addition to ref assemblies to get xml docs -->
     <ReferenceImplementationAssemblies>true</ReferenceImplementationAssemblies>
+    <ResolveAssemblyReferencesSilent>true</ResolveAssemblyReferencesSilent>
```

Comment:
Did you try the MSBuildWarningsAsMessages for the specific warning? I think that's less of a hammer in case something else is warned and is a real issue.

Reply:
You should not be suppressing this at all, doing so can introduce dangerous closure/assembly mismatch issues in your references which leak to customers.

Reply:
This is suppressed in this project that builds the ref pack, not reference assembly projects. The ref pack project references everything to harvest xml from impl, rest from ref. This would not suppress an error of ref referencing impl .

Reply:
In that case just turn it all off.  It's not serving any purpose, right?

Reply:
I haven't had time to study it all, but it is actually using RAR results. Potentially the implementation projects could be ReferenceOutputAssembly="false" or sequenced some other way, and just dig for the xml by convention or something?

Reply:
In that case I'd explicitly funnel whatever problematic items there are that are causing RAR trouble into a different ItemGroup that RAR doesn't look at.  If you get any warnings from RAR I wouldn't trust it to be doing what you want it to do.